### PR TITLE
OPENMP implementation with main rewrite in C

### PR DESCRIPTION
**Pull request by wkerzendorf**
Maybe interesting for @orbitfold and @ssim 
This will be cleaned up more before merge (but the general structure is there).

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -43,7 +44,7 @@
 builtins._ASTROPY_PACKAGE_NAME_ = PACKAGENAME
 
 # VERSION should be PEP386 compatible (http://www.python.org/dev/peps/pep-0386)
-VERSION = '1.0.dev'
+VERSION = '1.0.0.dev'
```
Comment (wkerzendorf) [inline setup.py]: remove last zero.

**Comment by wkerzendorf** [comment]
@mklauser is working on rescaling troubles

**Comment by wkerzendorf** [comment]
Correction: @unoebauer is figuring this out.

**Comment by ssim** [comment]
see #338 - closing this one.

**Pull request closed without merging**